index.js

```diff
@@ -1 +1,188 @@
 console.log("Hello!!!");
+
+//Return total days count
+//getDaysInYear(year);
+//getDaysInYear(2021);
+// 365
+//getDaysInYear(2020);
+// 366
+
+const getDaysInYear = (year) => {
+  let startDate = new Date(0);
+  const newDate = new Date(startDate.setFullYear(year))
+  const yearTime = startDate.setFullYear(year+1) - newDate;
+  const days = yearTime / 86400000;
+  return days
+}
+
+console.log(getDaysInYear(2021));
+console.log(getDaysInYear(2020));
+
+//Return day number from date
+//getDayNumber(date);
+//getDayNumber("2023-01-12");
+// 12
+//getDayNumber("2023-02-26");
+// 57
+
+const getDayNumber = (date) => {
+  const startDate = new Date(date);
+  const firstDayInYear = new Date(startDate);
+  firstDayInYear.setMonth(0);
+  firstDayInYear.setDate(1);
+  const dayNumber = ((startDate - firstDayInYear) / 86400000) + 1;
+  return dayNumber
+}
+
+console.log(getDayNumber("2023-01-12"));
+console.log(getDayNumber("2023-02-26"));
+
+// Return quarter number from date
+// I quarter: Jan - Mar
+// II quarter: Apr - Jun
+// III quarter: Jul - Sep
+// IV quarter: Oct - Dec
+// getQuarters(date);
+// getQuarters("2023-02-26");
+// Feb 26 is I quarter
+
+const getQuarters = (date) => {
+  const startDate = new Date(date);
+  const monthList = ['Jan','Feb','Mar','Apr','May','Jun','Jul','Aug','Sep','Oct','Nov','Dec'];
```

Comment:
it's better to have this type of data outside of function
because every time you call a function you create this arr and quarterlist and since you don't change them it's better to move out of function